src/server.ts

```diff
@@ -25,6 +26,12 @@ initializeMiddleware(app)
 
 const { routes, routePaths } = getRoutes()
 
+// Pre-emptive redirects
+//
+// These selectively match routes that would otherwise be handled by
+// the route matchers below
+app.get("/collection/:slug", redirectCollectionToArtistSeries)
```

Comment:
https://github.com/artsy/force/blob/main/src/Apps/Redirects/redirectsServerRoutes.tsx is maybe a more appropriate place

Reply:
Yep that was my first approach.

But I found that because that middleware is [loaded after](https://github.com/artsy/force/blob/c127c7b32c77ffd4557dd90267cceba2724b3a2d/src/server.ts#L68) the [regular route matchers](https://github.com/artsy/force/blob/c127c7b32c77ffd4557dd90267cceba2724b3a2d/src/server.ts#L35-L37) that I was therefore not able to match the `/collection/foo` paths that I needed to intercept (which were already getting matched by [the Collection app](https://github.com/artsy/force/blob/c127c7b32c77ffd4557dd90267cceba2724b3a2d/src/Apps/Collect/collectRoutes.tsx#L63))

That was why I hoisted this app and added the comment. Lmk if you think there is still a better way to do this.

Reply:
Oh! I see. Nice catch